dev/tasks/r/github.packages.yml

```diff
@@ -178,18 +231,13 @@ jobs:
         with:
            working-directory: 'arrow'
            extra-packages: cpp11
-      - name: Install sccache
-        if: startsWith(matrix.platform, 'macos')
-        run: brew install sccache
       - name: Build Binary
         id: build
         shell: Rscript {0}
         env:
-          NOT_CRAN: "true" # actions/setup-r sets this implicitly
+          NOT_CRAN: "false" # actions/setup-r sets this implicitly

```

Comment:
I have set this to false to make sure that we don't use the short cuts that setting it allows to be close in behavior to cran.

Reply:
Is it possible (and if so, is it better to) unset this instead of marking it as false? IIUC CRAN has this unset

Reply:
IIRC unsetting it in R isn't enough that's why I did this to be explict as our code needs it to be true to use the shortcuts.  From a technical perspective there is no way to unset it using the gha syntax https://github.com/actions/runner/issues/1126